variants/aws-k8s-1.19/Cargo.toml

```diff
@@ -15,6 +15,7 @@ kernel-parameters = [
     "console=ttyS0,115200n8",
     "net.ifnames=0",
     "netdog.default-interface=eth0:dhcp4,dhcp6?",
+    "quiet",
```

Comment:
NIT: can we skip this change? The variant will be deprecated by the next month:

#2228